transportable-udfs-examples/build.gradle

```diff
@@ -63,6 +63,8 @@ subprojects {
 
     checkstyle {
       configFile = file("${rootDir}/../gradle/checkstyle/linkedin-checkstyle.xml")
+      configProperties = ['config_loc' :  file("${rootDir}/../gradle/checkstyle")]
```

Comment:
Consider using the [`configDirectory`](https://docs.gradle.org/current/dsl/org.gradle.api.plugins.quality.CheckstyleExtension.html#org.gradle.api.plugins.quality.CheckstyleExtension:configDirectory) property.